examples/terraform/hetzner/main.tf

```diff
@@ -34,7 +34,7 @@ resource "hcloud_network_subnet" "kubeone" {
 resource "hcloud_server_network" "control_plane" {
   count       = 3
   server_id  = element(hcloud_server.control_plane.*.id, count.index)
-  network_id = hcloud_network.net.id
+  subnet_id = hcloud_network_subnet.kubeone.id
 }
```

Comment:
Nit: can you please run `terraform fmt`?

Reply:
done